pandas/tests/indexes/interval/test_interval_tree.py

```diff
@@ -207,3 +207,21 @@ def test_interval_tree_error_and_warning(self):
         ):
             left, right = np.arange(10), [np.iinfo(np.int64).max] * 10
             IntervalTree(left, right, closed="both")
+
+    @pytest.mark.skipif(not IS64, reason="GH 23440")
```

Comment:
Prefer `xfail` over `skipif` just in case this gets fixed in the future

Reply:
You mean like this?
```
@pytest.mark.xfail(not IS64, strict=False, reason="GH 23440")
```

That would be inconsistent with the other `skipif`s in the file. So maybe we should change all of them?